middle_end/flambda2.0/from_lambda/closure_conversion.ml

```diff
@@ -14,7 +14,7 @@
 (*                                                                        *)
 (**************************************************************************)
 
-[@@@ocaml.warning "+a-4-30-40-41-42"]
+[@@@ocaml.warning "+a-4-30-40-41-42-66"]
```

Comment:
Would you rather add that warning to the ignored ones (unused `!open`) or remove the `open! Int_replace_polymorphic_compare` ?
`!open` is only used for this, so I would rather be for silencing it and keeping the open just in case.